app/views/mediated_pages/_status_header.html.erb

```diff
@@ -1,3 +1,5 @@
+<% content_for :page_title do %>Request status: <%= current_request.item_title %><% end %>
```

Comment:
I believe this would only affect the status page for `MediatedPages`.

I think a more appropriate place for this would be in the [`status.html.erb`](https://github.com/sul-dlss/sul-requests/blob/master/app/views/requests/status.html.erb) view as that renders the `status_header` partial (which can be overridden and customized for different requests types).